<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PvarMenu</class>
 <widget class="QWidget" name="PvarMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set parameters which will not change throughout each simulation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton">
       <property name="text">
        <string>â€”</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="infoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value 2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionShow_Cells">
   <property name="text">
    <string>Show Cells</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>PvarMenu</receiver>
   <slot>on_addButton_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>PvarMenu</receiver>
   <slot>on_refreshButton_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoButton</sender>
   <signal>clicked()</signal>
   <receiver>PvarMenu</receiver>
   <slot>on_infoButton_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>PvarMenu</receiver>
   <slot>on_removeButton_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
